setup.py

```diff
@@ -64,7 +64,7 @@
 
 runtime_library_dirs = ["$ORIGIN/."] if not SYSTEM_LIBSSH else None
 _lib_dir = os.path.abspath("./src/lib") if not SYSTEM_LIBSSH else "/usr/local/lib"
-include_dirs = ["libssh/include"] if (ON_WINDOWS or ON_RTD) or \
+include_dirs = ["libssh/include", "./src/include"] if (ON_WINDOWS or ON_RTD) or \
```

Comment:
Is this change needed for something? `libssh/include` should already have the correct include file.

Reply:
CMake is now used to generate the libssh_version.h which is placed in the build tree. I suspect a cleaner way to handle this is to do a `cmake --install src --prefix local` after the build and then add `local/include` and `local/lib` as include/library directives. This way, the ssh-python build becomes decoupled from what the libssh CMake file is doing exactly.

Reply:
Makes sense, thanks for making the change.